app/workers/publish_worker.rb

```diff
@@ -30,6 +30,6 @@ def print_limitations_of_publishing(document)
     message = "Skipped publishing document with content_id #{content_id}"
     message += " because it has a state of '#{state}'."
 
-    puts message
+    puts message # rubocop:disable Rails/Output
```

Comment:
```suggestion
    logger.warn(message)
```

stretching the change a bit, but I imagine the puts here causes errors in logit

Reply:
I tried this originally, but unfortunately it's [covered by a dodgy test](https://github.com/alphagov/specialist-publisher/blob/master/spec/workers/publish_worker_spec.rb#L38). Seemed a bit much to start changing that. I guess we could do `expect(Rails.logger).to receive(:warn)`. What do you think?

Reply:
Ah darn. That's a good idea. I wonder if since the it isn't related to the logging that we actually need a test for it and skip that part. I imagine that was mostly put in because it added junk to the test output.

Reply:
Oh yes, I wasn't looking hard enough - removed that now.